### DISCUSSION TITLE
🎨 Ajouter un micro module de retour sur les champs d'autocomplétions

### DISCUSSION THREAD

**Discussion by wiinxt** [betagouv/mon-entreprise#2738] [Idées]
### Description de la proposition

Afin d'améliorer les résultats de ces champs :
- [ ] taux AT/MP 
- [x] code APE

Ce micro module serait sous la forme d'un bouton qui ouvre une popup/tooltip avec : 
- mots recherché qui ne donne pas de bon résultats
- taux/code qui devrait correspondre (optionnel)
- email pour le contacter au cas ou (et réduire les abus)

**Comment by johangirod** [comment]
On a testé sur code APE, on a reçu trop de demande, souvent non qualifiée. A réfléchir sur comment faire pour éviter d'avoir trop de demande à traiter. Champs avec juste les mots clés ?
